.github/workflows/ci.yml

```diff
@@ -50,7 +50,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions-rs/toolchain@v1
       with:
-        toolchain: 1.68.2
+        toolchain: stable
```

Comment:
I'd prefer a specific version, so we don't potentially break when the "stable" version updates.